pkg/util/k8sutil/backup.go

```diff
@@ -184,9 +184,8 @@ func NewBackupPodSpec(clusterName, account string, sp spec.ClusterSpec) (*v1.Pod
 				Name:  "backup",
 				Image: BackupImage,
 				Command: []string{
-					"/bin/sh",
-					"-ec",
-					"/usr/local/bin/etcd-backup --etcd-cluster=" + clusterName,
+					"/usr/local/bin/etcd-backup",
+					"--etcd-cluster=" + clusterName,
```

Comment:
That's fine. We can change the commands of those small jobs later.
This PR is mainly to change etcd operator's commands.